taccsite_cms/static/site_cms/css/src/_imports/components/django.cms.blog.app.css

```diff
@@ -0,0 +1,67 @@
+/*
+Django CMS Blog App Components
+
+These styles are only for the blog app pages (list pages, article pages).
+
+Reference:
+
+- [Django CMS Blog App Templates](https://github.com/nephila/djangocms-blog/tree/1.1.1/djangocms_blog/templates/djangocms_blog)
+
+Styleguide Components.DjangoCMS.Blog.App
+*/
+@import url("@tacc/core-styles/source/_imports/components/bootstrap.pagination.css");
```

Comment:
**Hint:** The [imported stylesheet](https://github.com/TACC/Core-Styles/pull/16/files#diff-2b5809c4a361283f83227b9b2c8e92f332b9a728a1a08ae366816eff5a3b44bc) comes from related PR https://github.com/TACC/Core-Styles/pull/16.